packages/amplify_api/android/build.gradle

```diff
@@ -65,10 +65,8 @@ dependencies {
     api amplifyCore
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.amplifyframework:aws-api:1.24.1"
-    implementation "com.amplifyframework:aws-api-appsync:1.24.1"
-    implementation 'androidx.test.ext:junit-ktx:1.1.3'
```

Comment:
This is needed, although probably just as a `testImplementation`

Reply:
Done!